Microsoft.Toolkit.Uwp.UI.Controls/PullToRefreshListView/PullToRefreshListView.xaml

```diff
@@ -40,8 +40,8 @@
                         <Grid>
                             <Border x:Name="RefreshIndicator" VerticalAlignment="Top">
                                 <Grid>
-                                    <TextBlock x:Name="DefaultIndicatorContent" HorizontalAlignment="Center" FontSize="20"></TextBlock>
-                                    <ContentPresenter Content="{TemplateBinding RefreshIndicatorContent}"></ContentPresenter>
+                                    <ContentPresenter x:Name="PullAndReleaseIndicatorContent" HorizontalAlignment="Center" FontSize="20" />
```

Comment:
Yes, might as well. Ideally, we use the Xaml Styler visual studio add-in for doing this (with default settings).